replicated-security/duality-1/README.md

```diff
@@ -0,0 +1,93 @@
+
+* **Chain-ID**: `duality-1`
+* **minimum-gas-prices**: ``
+* **timeout_commit**: `2s`
+* **Spawn time**: `2023-05-31T18:30:00Z`
+* **GitHub repo**: [duality-labs/duality](https://github.com/duality-labs/duality.git)
+* **Release**: [`v0.2.2`](https://github.com/duality-labs/duality/releases/tag/v0.2.2)
+<!-- * **Genesis file with CCV state:** [duality-1-genesis.json](duality-1-genesis.json) -->
+
+* **Reference binary**: [dualityd-linux-amd64](./dualityd-linux-amd64)
+* **Binary sha256sum**: `054b8cf641d7e9877048896ebe08343c1e95f6a8e7c3a0be044b91401f558099`
+* **Genesis file _without CCV state_:** [duality-1-genesis-without-ccv.json](duality-1-genesis-without-ccv.json), verify with `shasum -a 256 duality-1-genesis-without-ccv.json`
+* **SHA256 for genesis file _without CCV state_**: `f11efd315b7ef51ec9b28dc65e65c7d48280811823f718f4c13f5512a40ce99f`
+
+
+* Genesis file hash
+  * The SHA256 is used to verify against the genesis file (without CCV state) that the proposer has made available for review.
+  * The `duality-1-genesis-without-ccv.json` file cannot be used to run the chain: it must be updated with the CCV (Cross Chain Validation) state after the spawn time is reached.
+  * The genesis file includes funds for a relayer and a faucet account as well as account with funds for different internal needs, `signed_blocks_window` has been set to `864000`, and `min_signed_per_window` has been set to `5%`.
+* Binary hash
+  * The `dualityd-linux-amd64` binary is only provided to verify the SHA256. It was built with Interchain Security release [`v1.2.0-multiden`](https://github.com/cosmos/interchain-security/releases/tag/v1.2.0-multiden). You can generate the binary following the build instructions in that repo or using one of the scripts provided here.
+* Spawn time
+  * Even if a proposal passes, the CCV state will not be available from the provider chain until after the spawn time is reached.
+
+> If you want to build genesis by your own please take into account difference between the `ccvconsumer` section generated by version v9.0.2 of `gaiad` and the `ccvconsumer` section required for duality to run. Please check and add `"soft_opt_out_threshold": "0.05"` to the `params` section if it absent.
+
+For more information regarding the consumer chain creation process, see [CCV: Overview and Basic Concepts](https://github.com/cosmos/ibc/blob/main/spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md).
+
+## Endpoints
+
+Endpoints are exposed as subdomains for the sentry and snapshot nodes (described below) as follows:
+
+* TBD
+
+Seed nodes:
+
+1. TBD
+
+The following state sync node serve snapshots every 1000 blocks:
+
+1. TBD
+
+## How to Join
+
+### Hardware Requirements
+
+* 4 Cores
+* 32 GB RAM
+* 2x512 GB SSD
+
+### Software Versions
+
+| Name               | Version  |
+|--------------------|----------|
+| Duality            | v0.2.2   |
+| Go                 | > 1.18   |
```

Comment:
We've seen strange issues when validators use different Go versions to build the binaries. @teddyknox can you recommend a specific Go version here?

Reply:
Sure, going with the version installed on my machine.